src/accounts/solana.ts

```diff
@@ -5,16 +5,30 @@ import * as solanajs from "@solana/web3.js";
 import nacl from "tweetnacl";
 import base58 from "bs58";
 
+export interface SolanaWallet {
+    signMessage(message: Uint8Array): Promise<Uint8Array>;
+    publicKey: solanajs.PublicKey;
+}
```

Comment:
This interface is already defined in the official [Solana WalletAdapter](https://solana-labs.github.io/wallet-adapter/) package.